src/mono/mono/mini/mini-arm64.c

```diff
@@ -6240,6 +6329,16 @@ mono_arch_emit_epilog (MonoCompile *cfg)
 		break;
 	}
 
+#ifdef MONO_ARCH_HAVE_SWIFTCALL
+	if (mono_method_signature_has_ext_callconv (cfg->method->signature, MONO_EXT_CALLCONV_SWIFTCALL)) {
+		if (cfg->arch.swift_error_var) {
```

Comment:
The ifdef and the if is not needed, its enough to check cfg->arch.swift_error_var ?
Also, shouldn't this be done after the native call to the swift function instead of in the epilog ?

Reply:
It is done in the wrapper epilog which is after the native call. Is there a better place to do it?

Reply:
Thanks, moved to `emit_move_return_value`.